### ISSUE TITLE
WCAG 1.3.1: <li> elements must be contained in a <ul> or <ol> (.py-0.Box-row:nth-child(3))

### ISSUE THREAD

**Issue by AhmedAbdoOrtiga** [AhmedAbdoOrtiga/Public_Rule_Ortiga#3]
**Issue**: `<li> elements must be contained in a <ul> or <ol>` ([`listitem`](https://dequeuniversity.com/rules/axe/3.1/listitem?application=axeAPI))

**Target application**: [Options](https://github.com/AhmedAbdoOrtiga/accessibility-insights-windows/settings)

**Element path**: ul[role="group"] > .py-0.Box-row:nth-child(3)

**Snippet**:

    <li class="Box-row py-0">

**How to fix**:

    Fix any of the following:
      List item does not have a <ul>, <ol> or role="list" parent element

**Environment**:
Chrome version 72.0.3626.119

====

This accessibility issue was found using Accessibility Insights for Web 1.0.4 (axe-core 3.1.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
